doc/start/quick-ceph-deploy.rst

```diff
@@ -40,13 +40,10 @@ Create a Cluster
 If at any point you run into trouble and you want to start over, execute
 the following to purge the configuration::
 
+	ceph-deploy purge {ceph-node} [{ceph-node}]
```

Comment:
Hh～, that is better. I will fix it soon.

And could you know why 'cmake'  can't compile <pthread> or <venctor> and some other head file when I have installed all dependences. Something wrong in Makefile ?
OS : centos7
cmake:2.8.11
Please teach  me, I have been troubled by this for a long time。Thanks!